parm/use_cases/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed.conf

```diff
@@ -0,0 +1,194 @@
+# LEAD_SEQ is the list of forecast leads to process
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#timing-control
+###
+
+LOOP_BY = INIT
+INIT_TIME_FMT = %Y%m%d%H
+INIT_BEG = 2022070108
+INIT_END = 2022070108
+#INIT_BEG = 2022070701
+#INIT_END = 2022073123
+INIT_INCREMENT = 1H
+
+LEAD_SEQ = 12
+#LEAD_SEQ = 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18
+
+# USER SETTINGS HERE
+# CUSTOM_LOOP_LIST = DENVER, DALLAS, BOSTON, MINNEAPOLIS   [list of airports sepearated by commas; each needs a mask file]
+# PY_SOUNDING_FLAG = ALL, ASC, DESC  [only one value here]
+# PY_PT_DELTA =  Potential Temperature delta setting (K) [usually 1.0-2.5]
+# Valid_Time(YYYYMMDD_HHMMSS)
+CUSTOM_LOOP_LIST = DENVER
+PY_SOUNDING_FLAG = ALL  
+PY_PT_DELTA = 1.25
+PY_VAL_TIME = {valid?fmt=%Y%m%d_%H0000}
+
+###
+# File I/O
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#directory-and-filename-template-info
+###
+
+GEN_VX_MASK_INPUT_DIR = {INPUT_BASE}/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed/{init?fmt=%Y%m%d%H}
+GEN_VX_MASK_INPUT_TEMPLATE = pblh_22{init?fmt=%j}{init?fmt=%H}0000{lead?fmt=%HH}
+GEN_VX_MASK_INPUT_MASK_DIR = {INPUT_BASE}/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed
+GEN_VX_MASK_INPUT_MASK_TEMPLATE = met_mask_{custom?fmt=%s}.txt
+
+GEN_VX_MASK_OUTPUT_DIR = {INPUT_BASE}/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed
+GEN_VX_MASK_OUTPUT_TEMPLATE = {OUTPUT_BASE}/gen_vx_mask_pblh/met_mask_{custom?fmt=%s}_100km.nc
+
+FCST_POINT_STAT_INPUT_DIR = {INPUT_BASE}/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed
+FCST_POINT_STAT_INPUT_TEMPLATE = {init?fmt=%Y%m%d%H}/pblh_22{init?fmt=%j}{init?fmt=%H}0000{lead?fmt=%HH}
+
+OBS_POINT_STAT_INPUT_DIR =
+OBS_POINT_STAT_INPUT_TEMPLATE = PYTHON_NUMPY= {PARM_BASE}/use_cases/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed/calc_amdar_pblh.py {INPUT_BASE}/model_applications/pbl/PointStat_fcstHRRR_obsAMDAR_PBLH_PyEmbed/22{valid?fmt=%j}{valid?fmt=%H}00q.cdf
+
+POINT_STAT_OUTPUT_DIR = {OUTPUT_BASE}/point_stat_pblh
+POINT_STAT_OUTPUT_TEMPLATE = {valid?fmt=%Y%m%d}
+
+POINT_STAT_CLIMO_MEAN_INPUT_DIR =
+POINT_STAT_CLIMO_MEAN_INPUT_TEMPLATE =
+
+POINT_STAT_CLIMO_STDEV_INPUT_DIR =
+POINT_STAT_CLIMO_STDEV_INPUT_TEMPLATE =
+
+
+###
+# Field Info
+# https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#field-info
+###
+
+POINT_STAT_ONCE_PER_FIELD = False
+
+#POINT_STAT_FCST_FILE_TYPE =
+#POINT_STAT_OBS_FILE_TYPE =
+
+FCST_VAR1_NAME = HPBL
+FCST_VAR1_LEVELS = L0
+FCST_VAR1_THRESH = <=0, >10000
+OBS_VAR1_NAME = HPBL
+OBS_VAR1_LEVELS = L0
+OBS_VAR1_THRESH = <=0, >10000
+OBS_VAR1_OPTIONS = set_attr_units = "m"
+
+
+###
+# GenVxMask Settings
+# https://metplus.readthedocs.io/en/latest/Users_Guide/wrappers.html#genvxmask
+###
+
+GEN_VX_MASK_SKIP_IF_OUTPUT_EXISTS = True
+GEN_VX_MASK_OPTIONS = -type "circle" -thresh le100
```

Comment:
Under `#USER SETTINGS HERE` at the top of the file, you may consider defining a local configuration variable called "GEN_VX_MASK_AIRPORT_RADIUS_KM" or something like that, and then reference it in the following items:
```
GEN_VX_MASK_OPTIONS
GEN_VX_MASK_OUTPUT_TEMPLATE
POINT_STAT_MASK_POLY
```

So that if you or another user desires a wider or narrower radius they just change the value one time and everything else updates.

Reply:
I've added that